final/src/components/Fields/Datepicker.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import DatePicker from 'react-datepicker';
+import 'react-datepicker/dist/react-datepicker.css';
+
+export const DatePickerField = ({ 
+  dateFormat, 
+  type, 
+  input, 
+  maxDate,
+  minDate,
+}) => {
+  return (
+    <DatePicker
+      { ...input }
+      type={ type }
+      selected={ new Date(input.value) }
+      dateFormat={ dateFormat }
+      maxDate={ maxDate }
+      minDate={ minDate }
+    /> 
+  );
+};
+
+DatePickerField.propTypes = {
```

Comment:
Wrong prop types specified. iFor example input is specified as string, but it should be object instead.